src/Bicep.Core/SourceGraph/ArmTemplateFile.cs

```diff
@@ -10,7 +11,9 @@ namespace Bicep.Core.SourceGraph
 {
     public class ArmTemplateFile : ISourceFile
     {
-        public ArmTemplateFile(Uri fileUri, IFileHandle fileHandle, string text, Template? template, JObject? templateObject, IFeatureProvider featureProvider)
+        private readonly IConfigurationManager configurationManager;
```

Comment:
We shouldn't have ARM template file depending on Bicep config - instead we need to represent any information required by Bicep directly in the template.

Reply:
Removed.